quickstart/gcp-cloudshell-quickstart.py

```diff
@@ -242,6 +243,15 @@ def ssh_key_create(path):
     subprocess.run(ssh_cmd.split(' '), check=True)
 
 
+def cam_deployment_sa_create_key(service_account, filepath):
```

Comment:
this is just writing a dictionary to a file, don't think we need a custom function for just 2 lines.